libobs/obs-source.c

```diff
@@ -479,26 +485,39 @@ static void duplicate_filters(obs_source_t *dst, obs_source_t *src,
 			obs_source_duplicate(src_filter, new_name, private);
 		obs_source_set_enabled(dst_filter, enabled);
 
-		bfree(new_name);
+		if (recordNames) {
+			new_names[i - 1] = new_name;
+		} else {
+			bfree(new_name);
+		}
+
 		obs_source_filter_add(dst, dst_filter);
 		obs_source_release(dst_filter);
 		obs_source_release(src_filter);
 	}
 
 	da_free(filters);
+
+	return new_names;
 }
 
 void obs_source_copy_filters(obs_source_t *dst, obs_source_t *src)
+{
+	obs_source_copy_filters2(dst, src, false);
```

Comment:
There is no third parameter for `obs_source_copy_filters2`.  This causes a build failure on Ubuntu.
```
/home/runner/work/obs-studio/obs-studio/libobs/obs-source.c: In function ‘obs_source_copy_filters’:
/home/runner/work/obs-studio/obs-studio/libobs/obs-source.c:506:2: error: too many arguments to function ‘obs_source_copy_filters2’
  obs_source_copy_filters2(dst, src, false);
  ^~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Yeah, this is the error I was talking about. I am unable to compile because of this.

Reply:
I removed it. (I'm surprised I could compile it on Windows before filing PR.)